build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
```

Comment:
Is this change necessary for some lib?

Reply:
This connected with new version of Android Studio 3.1.1. I think we should support its actual state